examples/qsp_vlasov.py

```diff
@@ -73,81 +71,47 @@ def main():
         tolerable_logical_error_rate / 3
     )  # Allocate half the error budget to trotter precision
 
-    gate_synthesis_error_budget = (
-        tolerable_logical_error_rate - qsp_required_precision
-    ) / 3
-    error_correction_error_budget = (
-        tolerable_logical_error_rate - qsp_required_precision
-    ) / 3
+    error_budget = {
+        "qsp_required_precision": qsp_required_precision,
+        "tolerable_circuit_error_rate": tolerable_logical_error_rate,
+        "total_error": 1e-2,
+        "synthesis_error_rate": 0.5,
+        "ec_error_rate": 0.5,
+    }
+    specs = {
```

Comment:
I think we're not using these specs anywhere in the end – instead we use transformers. Right?